x-pack/plugins/security_solution/server/lib/detection_engine/routes/rules/utils.test.ts

```diff
@@ -466,74 +427,6 @@ describe.each([
     });
   });
 
-  describe('transformOrImportError', () => {
```

Comment:
Looks like this was just dead code? 👍🏽 🧹

Reply:
That is correct, I double check for dead code. I almost think we should look for a way to auto-detect and sweep through all dead code and remove it. Devs spend a lot of time maintaining dead code which doesn't make sense to me. We should just remove it automatically through lint or tooling if we can.

I assume it's tricky in a few instances but if we could somehow review all of it at once on a cadence that would be super helpful and healthy.